prql-compiler/Cargo.toml

```diff
@@ -65,8 +67,8 @@ rusqlite = {version = "0.28.0", features = ["bundled", "csvtab"]}
 
 # Re-enable on windows when duckdb supports it
 # https://github.com/wangfenjin/duckdb-rs/issues/62
-# [target.'cfg(not(any(target_family = "windows", target_family="wasm")))'.dev-dependencies]
-# duckdb = {version = "0.5.0", features = ["bundled", "chrono"]}
+[target.'cfg(not(any(target_family = "windows", target_family="wasm")))'.dev-dependencies]
```

Comment:
Ah, I forgot about this. We can ignore this PR if this still causes an issue for you @aljazerzen

Reply:
Whoops, this shouldn't be on semantic branch, I reverted the change with ed036e8.